app/mailers/employee_invitation_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class EmployeeInvitationMailer < ApplicationMailer
+  default from: 'test@test.com'
+
+  def invite_employee(email)
+    mail(to: email,
```

Comment:
Use the email as an instance variable here
```suggestion
  def invite_employee
    @email = params[:email]
  
    mail(to: @email,
```

Reply:
I'm already recieving this email as a instance variable when I pass it as a parameter to the method, should I replace it for `
@email = params[email]` or can I keep it this  way?